README.md

```diff
@@ -1 +1,32 @@
-# angular-hello-world-lab
\ No newline at end of file
+# Hello World 
+
+## Objectives
+
+- Create a Controller
+- Create an element for Controller binding
+- Create a public $scope method
+- Display the $scope value in the View
+
+## Instructions
+
+Setup the directory structure as follows:
+
+- js/
+  - app/
+    - controllers/
+  - angular.js
+- index.html
+
+You can find `index.html` and `angular.js` in this repo.
+
+Create a new module in `js/app/app.js`, named `app`.
+
+Create a controller named `MainController` inside `js/app/controllers/MainController.js` and attach it to a module named `app`. The function should take one parameter - `$scope`.
+
+In our HTML, initiate our module `app` using `ng-app` on a HTML element.
+
+Initiate our controller `MainController` using `ng-controller` on a HTML element inside our previous one.
+ 
+Inside `MainController`, assign some values to the `$scope` object.
```

Comment:
For labs, it's important to make the instructions as specific as possible. This is how we can reinforce specific learning objectives. Give the student a clear target and feel free to have some fun. 

In this case, it would be awesome if the students attached an object to the `$scope` like in your previous Bill Gates example and used those properties in the view. Maybe it's "Today, we're going to build out a Facebook profile page. Your `$scope` should have one property, `user`, which points to an object. The object should have properties for `name`, `email`, and `imageUrl`. Feel free to think of something else that's more fun. Either way, make sure that the requirement is specific and tied to the learning objective.